src/test/java/com/google/cloud/spanner/pgadapter/StatementTest.java

```diff
@@ -14,6 +14,12 @@
 
 package com.google.cloud.spanner.pgadapter;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
```

Comment:
Added static imports for commonly used mock and assertion methods. This is the main reason behind the large number of changes in this file, and most changes are only replacements of `Mockito.when(...)` with `when(...)` etc.